### DISCUSSION TITLE
Allow for manual submission of scores in the event that the client fails

### DISCUSSION THREAD

**Discussion by shikkarishirou** [ppy/osu#14324] [Ideas]
The majority of the player base has probably experienced something where client score submission failed or went wrong, such as trying to submit scores when the server was down, or maybe they live in an area where there isn't reliable internet (me), or maybe the client crashes during score submission. I feel like there should be a way to submit a score manually after a play, something like a button on the replay screen that when pressed tries to submit a score to the server. If proper anti-ddos measures are taken, such as strictly limiting how often an attempt can be made, I think there would be no real drawbacks to a feature like this and it would make the game a lot more enjoyable for many people.

**Comment by peppy** [comment]
Sorry but this is not feasible. The client will do its best to send your scores, but delayed sending of scores its not something we can support (score validity becomes an issue).
